src/data/repositories/DHISRepository.js

```diff
@@ -0,0 +1,134 @@
+import { apiUrl } from "../../webapp/app.js";
+
+export class DHISRepository {
+	get($resource, dataSetsIds) {
```

Comment:
@tokland I'm sorry but, how I can add the `$resource` to the constructor? I suppose that it should be added inside `compositionRoot` but I don't know how exactly...

Reply:
Exactly, in the compositionRoot you'd do `new DHISRepository($resource)`. Not sure if that's available when you call getCompositionRoot, though.

Reply:
I think it only can be referenced inside angular modules, because is required to load `ngResource` previously in the module to be able to use it. So encapsulation won't let use it inside the compositionRoot with export. Also is the old angularjs and not the new one.